README.md

```diff
@@ -19,54 +26,53 @@ go build ./cmd/kind-test
 
 ### Building the image
 
-Requires `gcloud` authenticated and configured.
+#### Using Gcloud
 
-Requires a google cloud project
+Make sure `gcloud` is authenticated and configured.
 
-`./scripts/publish-manager.sh`
+You also need to set up a google cloud project.
 
-#### Using Docker
+Run: `./scripts/publish-manager.sh`
 
-Alternatively, replace "my-repository" with an appropriate prefix and run:
+#### Using Docker
 
-```
-docker build -t my-repository/capd-manager:latest .
-```
+Alternatively, run: `docker build -t <MY_REPOSITORY>/capd-manager:latest .`
 
-# Testing out CAPD
+## Trying CAPD
 
 Tested on: Linux, works ok on OS X sometimes
 
-Requirements: `kind` > 0.3.0 and `kubectl`
+Make sure you have `kind` > 0.3.0 and `kubectl`.
 
-Install capdctl
+1. Install capdctl: 
```

Comment:
These don't count properly. i think the sub paragraphs need to be indented?

1. hello
    this is a test
1. hello
    still a test

yeah like this